examples/flutter_search/lib/models/entities.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:equatable/equatable.dart';
+
+class SearchResult extends Equatable {
+  final List<SearchResultItem> items;
+  final bool isPopulated;
```

Comment:
why do we need `isPopulated` and `isEmpty`? Can't that all be derived from `items`?

Reply:
What do you mean ? By doing this :
```dart
searchResultItems.length == 0 //isEmpty
```
I just copied an paste your angular example model but my suggestion would be to discard one of these two booleans because they reflect the same information.

Reply:
yeah exactly my thinking. Since it's the same in the angular example I can update both separately.